circle.yml

```diff
@@ -66,6 +74,10 @@ commands:
     steps:
       - attach_workspace:
           at: ~/
+      - when:
```

Comment:
I intended to have the newest Chrome not to be installed for the server-e2e-tests-electron-*.  install-latest-chrome is still installed for the electron tests. :(  I could add a boolean flag to dictate whether the browser is installed or not.  Is there a better way?

Reply:
I wonder if you could update the `install-latest-chrome` command to exit if the supplied `browser` is not `chrome`, maybe something like:

```sh
if [[ << parameters.browser >> = 'chrome' ]]
then
	echo 'Running Chrome tests, installing latest version...'
else
	echo 'Running in << parameters.browser >>, not updating Chrome...'
	exit 0
fi
```